osgtest/tests/test_190_condorce.py

```diff
@@ -34,31 +33,41 @@ def test_02_reconfigure_condor(self):
 
         command = ('condor_reconfig', '-debug')
         core.check_system(command, 'Reconfigure Condor')
-        self.assert_(service.is_running('condor', timeout=10), 'Condor not running after reconfig')
+        self.assertTrue(service.is_running('condor', timeout=10), 'Condor not running after reconfig')
+
+    def test_02_scitoken_mapping(self):
+        core.state['condor-ce.wrote-mapfile'] = False
+        core.skip_ok_unless_installed('condor', 'htcondor-ce')
```

Comment:
I like this suggestion but I don't think it's quite that easy since some of the tests set `core.state` key-val pairs before checking for the installed packages. `core.state` should probably be a `defaultdict(lambda: False)` but I think that's a bigger change than what's warranted in this PR so I'm going to go ahead with the merge for now.